dwave/optimization/include/dwave-optimization/utils.hpp

```diff
@@ -166,4 +167,23 @@ void deduplicate_diff(std::vector<Update>& diff);
 // Return whether the given double encodes an integer.
 bool is_integer(const double& value);
 
+// Forward declaration
+class ArrayNode;
```

Comment:
I don't see where this is used.

Reply:
It's not, I had used it in a previous version of `is_variant`. Will remove.